tests/cli/workflow/balsamic/test_cli_balsamic_config_case.py

```diff
@@ -22,7 +22,7 @@ def test_without_options(cli_runner: CliRunner, balsamic_context: CGConfig):
 
 
 def test_with_missing_case(cli_runner: CliRunner, balsamic_context: CGConfig, caplog):
```

Comment:
type hint `caplog: LogCaptureFixture` and the same for the functions below

Reply:
Changed ☺️